src/Uno.UI/Controls/CommandBarNavigationItemRenderer.iOS.cs

```diff
@@ -217,11 +221,17 @@ protected override Size MeasureOverride(Size availableSize)
 						&& !_childSize.Height.IsNaN()
 						&& _childSize.Height != 0
 						&& _childSize.Width != 0
-						&& (Frame.Width != _childSize.Width
-						|| Frame.Height != _childSize.Height))
+						&& ( (Frame.Width != _childSize.Width && _childSize.Width < Frame.Width)
```

Comment:
I dunno if this will cause problems, but the good thing is you're adding lots of tests :D